website/docs/docs/getting-started/2-your-first-test.md

```diff
@@ -0,0 +1,41 @@
+---
+id: your-first-test
+title: Writing your first test
+---
+
+To demonstrate how you can write tests, let's install the TodoMVC app from BigTest:
+```
+$ yarn add -D @bigtest/todomvc
+```
+
+Assuming you went with the default settings for `bigtest.json`, the app command should be configured as `yarn start`:
+```json
+{
+  "app": {
+    "command": "yarn start",
+    "url": "http://localhost:3000"
+  }
+}
+```
+Go into your `package.json` file and modify the start script as `yarn bigtest-todomvc 3000`:
```

Comment:
I think I'm lost here, but perhaps because I don't know to whom this package.json belongs to. Might be clarified from the question above of where this is happening.

Reply:
Okay so to give a better idea of what I was trying to do, I made some diagrams:

Say I have `Min's Website` and I want to try out bigtest. Normally I would install bigtest and configure it to my website. The red arrow indicates configuration.
![Screen Shot 2020-11-09 at 9 24 25 AM](https://user-images.githubusercontent.com/29791650/98553518-b053e580-226d-11eb-9931-5066058376db.png)

The idea is we want to guide them step by step as much as possible but if we were to make a guide that works for any app/website, we have to be more vague. We can't help them write test for a headings at the top of the website because some websites for some reason might not have a heading. So that's why we're slapping on the todomvc app on `Min's Website` so people can practice writing test directly from their project.

Another strategy would be to just have readers clone the todomvc app and write tests for it from the todomvc repo directly. But I think people want to dive into it as soon as possible. Meaning when there's a tool and the guide tells you to `git clone ourTools-tutorial.git`, I end up skipping it and just reading the documentation. But if the tutorial lets you put it on your project as soon as possible, it's more encouraging. We can discuss that theory further.

> The summary of the last two paragraphs: put it in your actual repo asap = "okay let's do this!". make people clone an example repo for practice = "... eh maybe i'll do it some other time. too much of a hassle."

So then when people add the todomvc app to their repo, it looks like this:
![Screen Shot 2020-11-09 at 9 25 25 AM](https://user-images.githubusercontent.com/29791650/98554430-b26a7400-226e-11eb-9580-a3ed631795bd.png)

Instead of configuring bigtest to Min's Website, we're just configuring it to todomvc app. And once people go through the guides, we can instruct them to remove todomvc app and fix the start script in their package.json to react-scripts start or ng serve or whatever it is they're using.

![Screen Shot 2020-11-09 at 9 26 23 AM](https://user-images.githubusercontent.com/29791650/98554644-f493b580-226e-11eb-8da9-30e039a3daba.png)

Reply:
It seems a bit counterintuitive to install an entire app in your app to try out a framework, but the rationale you provide is very strong. Perhaps we can just make clear from the beginning that that's the approach that we'll take, and even add a small note on the fact that they can also clone the project if it makes them feel more comfortable. 

I can try to write that down so you can focus on new content, I'm asking to understand what you want to say before I bring it to the editor and such. With this context that you provided now I can write out some more details on the guide. Thanks!